content/_index.md

```diff
@@ -36,7 +36,7 @@ I ran the transform on several sample images and they came out looking ok. I was
 
 Some code is a delight to work with. You know how to find what you need to work on. You know how to make the change you need. The code is easy to navigate, easy to understand, easy to reason about. You feel confident that your change will have the effect you wanted without any undue side-effects. The code *guides you*, invites you to look around. The programmer(s) who came before you cared about the person who would come along later, maybe because they realised that the programmer who came along later might be them!
 
-In his seminal book, "Refactoring", Martin Fowler says:
```

Comment:
I use the rule that punctuation goes inside quotes for a sentence or phrase, but not for a term, title, or fragment.

So:
- In "Refactoring", Martin says...
but
- I always say "Never look a gift horse in the mouth."